File: lib/platform/channels.dart
/// Channel names shared with the native Kotlin side. Keep in sync with
/// `android/app/src/main/kotlin/app/timeserver/bridge/Channels.kt`.
class Channels {
  Channels._();

  static const String ntpMethod = 'app.timeserver/ntp';
  static const String ntpStatusEvent = 'app.timeserver/ntp/status';
  static const String ntpLogsEvent = 'app.timeserver/ntp/logs';

  static const String gpsMethod = 'app.timeserver/gps';
  static const String gpsTimeEvent = 'app.timeserver/gps/time';

  static const String gnssEvent = 'app.timeserver/gnss/satellites';

  static const String powerMethod = 'app.timeserver/power';

  static const String networkMethod = 'app.timeserver/network';
  static const String networkEvent = 'app.timeserver/network/changes';

  static const String compassEvent = 'app.timeserver/compass';
}

File: android/app/src/main/kotlin/app/timeserver/bridge/Channels.kt
package app.timeserver.bridge

/** Channel names shared with the Dart side (see lib/platform/channels.dart). */
object Channels {
    const val NTP_METHOD = "app.timeserver/ntp"
    const val NTP_STATUS = "app.timeserver/ntp/status"
    const val NTP_LOGS = "app.timeserver/ntp/logs"

    const val GPS_METHOD = "app.timeserver/gps"
    const val GPS_TIME = "app.timeserver/gps/time"

    const val GNSS = "app.timeserver/gnss/satellites"

    const val POWER = "app.timeserver/power"

    const val NETWORK = "app.timeserver/network"
    const val NETWORK_EVENT = "app.timeserver/network/changes"

    const val COMPASS = "app.timeserver/compass"
}
